src/client/components/ConsentsSubHeader.tsx

```diff
@@ -78,8 +78,9 @@ const li = (numPages: number) => css`
     ${textSans.small()}
   }
   position: relative;
-  width: ${100 / numPages}%;
+  width: ${lastPage ? 'min-content' : 100 / (numPages - 1) + '%'};
```

Comment:
This makes it so that the last column width is as small as possible and we spread the remaining columns evenly.

We do this because we want to make component look visually centred. Without this modification, the furthest column would push the content further to the left than we want.